scripts/data/loadLandCover.R

```diff
@@ -25,59 +31,65 @@ library("gdalcubes")
 library("RCurl")
 options(timeout = max(60000000, getOption("timeout")))
 
-
+setwd(outputFolder)
 
 input <- fromJSON(file=file.path(outputFolder, "input.json"))
 print("Inputs: ")
 print(input)
 
 
 # Tranform the vector to a bbox object
+
 bbox <- sf::st_bbox(c(xmin = input$bbox[1], ymin = input$bbox[2], 
-            xmax = input$bbox[3], ymax = input$bbox[4]), sf::crs = st_crs(input$proj)) 
+                  xmax = input$bbox[3], ymax = input$bbox[4]), crs = sf::st_crs(input$srs_cube))
+
 
 n_year <- as.integer(substr(input$t1, 1, 4)) - as.integer(substr(input$t0, 1, 4)) + 1 
 temporal_res <- paste0("P", n_year, "Y")
 
 if (input$stac_source == "IO") {
   lc_raster <- stacatalogue::load_prop_values(stac_path = "https://io.biodiversite-quebec.ca/stac/",
-                                collections = c("esacci-lc"), 
+                                collections = "esacci-lc", 
```

Comment:
should use collections parameter

Reply:
No, that's not the point, you should use the parameter of the script! Something like input$collections

Reply:
Ok, changed hard coding for parameter collections =  "esacci-lc" to input$collections